src/Manager.sol

```diff
@@ -154,10 +154,10 @@ abstract contract Manager is
 
     // TODO: should we store this in storage? otherwise they can change during
     // each upgrade
-    address immutable _token;
-    Mode immutable _mode;
-    uint16 immutable _chainId;
-    uint256 immutable _evmChainId;
+    address public immutable _token;
```

Comment:
let's add a comment here to remind ourselves in the future that when adding new immutables, the `upgrade` function should be updated